tests/foreman/cli/test_activationkey.py

```diff
@@ -306,7 +306,7 @@ def test_negative_create_with_usage_limit_with_not_integers(self):
         :expectedresults: Activation key is not created. Appropriate error
             shown.
 
-        :CaseImportance: Critical
+        :CaseImportance: Low
```

Comment:
Tier1 is reserved for Functional tests on Component level with importance Critical, so this change should go with change to tier decorator. The same applies to many other tests in this file.

Reply:
moved to tier3